### ISSUE TITLE
Benchmark Stats

### ISSUE THREAD

**Issue by AlmasM** [ageitgey/face_recognition#46]
Hi,

I successfully installed your package. Thank you for providing convenient face recognition package. I wanted to inquire more about benchmark testing and accuracy. Do you have a paper or any additional references I could use for my research? Any statistical evidence or reference would be much appreciated.

Kind regards,

Almas M.

**Comment by ageitgey** [comment]
Since this library depends on the dlib implementation, the best description is this blog post: http://blog.dlib.net/2017/02/high-quality-face-recognition-with-deep.html

Here's where the description starts on that page (text written by Davis King, not me):

> Also, the model has an accuracy of 99.38% on the standard Labeled Faces in the Wild benchmark. This is comparable to other state-of-the-art models and means that, given two face images, it correctly predicts if the images are of the same person 99.38% of the time.

> For those interested in the model details, this model is a ResNet network with 29 conv layers. It's essentially a version of the ResNet-34 network from the paper Deep Residual Learning for Image Recognition by He, Zhang, Ren, and Sun with a few layers removed and the number of filters per layer reduced by half.

There's some more details on how the model was trained on that page (including a link to the training code).
